DSL/Pipelines/pipelines/opensearch.js

```diff
@@ -153,6 +153,20 @@ router.post('/delete/:index_name', (req,res) => {
 
 })
 
+router.post('/delete/object/:index_name', (req, res) => {
+	var index_name = req.params.index_name;
+	var obj_id = req.body.id;
+
+	osDeleteObject(index_name, obj_id).then((ret) => {
+		res.status(200);
+		res.end(JSON.stringify(ret));
+	}).catch((e) => {
+		res.status(500);
```

Comment:
I don't know the contents here, but is 500 the best code to return?

Reply:
It was 500 on all the other calls in the same file, so I wrote it with the same pattern. Maybe Alar knows a better answer.